data/json/npcs/godco/cook.json

```
@@ -0,0 +1,267 @@
+        "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "constant": -2 }, { "one_in": 4 } ] } ] }
+      },
+      { "skill": "cooking", "bonus": { "rng": [ 2, 4 ] } }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "GODCO_cook_worn",
+    "subtype": "collection",
+    "entries": [
+      { "item": "boots" },
+      { "item": "briefs" },
+      { "item": "socks" },
+      { "item": "sweatshirt" },
+      { "item": "pants" },
+      { "item": "jacket_chef" },
+      { "item": "leather_pouch" },
+      { "item": "hat_boonie", "chance": 66 },
+      { "item": "holy_symbol_wood" }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "GODCO_cook_carried",
+    "subtype": "collection",
+    "entries": [ { "item": "matches", "count": [ 12, 20 ] }, { "group": "snacks", "repeat": [ 1, 3 ] } ]
+  },
+  {
+    "type": "item_group",
+    "id": "GODCO_cook_wield",
+    "subtype": "collection",
+    "entries": [ { "item": "knife_steak" } ]
+  },
+  {
+    "type": "effect_type",
+    "id": "u_met_godco_cook"
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_cook_1",
+    "dynamic_line": {
+      "u_has_var": "u_met_godco_cook",
+      "type": "general",
+      "context": "meeting",
+      "value": "yes",
+      "yes": [ "Hey there.", "Hello.", "How are you?", "Welcome!", "How's the weather?" ],
+      "no": "Welcome!  You seem new, how can I help you?"
+    },
+    "responses": [
+      {
+        "text": "Who are you?",
+        "effect": { "u_add_var": "u_met_godco_cook", "type": "general", "context": "meeting", "value": "yes" },
+        "condition": { "not": { "u_has_var": "u_met_godco_cook", "type": "general", "context": "meeting", "value": "yes" } },
+        "topic": "TALK_GODCO_cook_firstmeet"
+      },
+      {
+        "text": "What is this place?",
+        "condition": { "u_has_var": "u_met_godco_cook", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_GODCO_cook_place"
+      },
+      {
+        "text": "What's your story?",
+        "condition": { "u_has_var": "u_met_godco_cook", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_GODCO_cook_ask_past"
+      },
+      {
+        "text": "How are things here?",
+        "condition": { "u_has_var": "u_met_godco_cook", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_GODCO_cook_ask_mood"
+      },
+      {
+        "text": "Can I do anything for you?",
+        "condition": { "u_has_var": "u_met_godco_cook", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_MISSION_LIST"
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_cook_firstmeet",
+    "dynamic_line": "Shouldn't I be the one to ask that question?  I am Simon, I work here as a cook.  Who are you?",
+    "responses": [
+      { "text": "I am a beggar who really needs something to eat.", "topic": "TALK_GODCO_cook_ask_food" },
```

Beggar look like slightly out of place here.
```suggestion
      { "text": "I am an unfortunate who really needs something to eat.", "topic": "TALK_GODCO_cook_ask_food" },
```